### PR TITLE
Replaced Python module with configparser

### LINKED ISSUE #13: Refactor config to use configparser
**`configparser`** is a widespread, simple-to-use module that's primarily used to create and manipulate configuration files.

Currently, one can use this bot by entering their Twitter credentials in a file called `credentials.py` in the [src](src) folder in a way that's slightly unorthodox and non-standard and may even be intimidating towards new contributors.

https://github.com/AlwaysLivid/TinyFlowerBeds/blob/856445843b986ff27a1619012332a2cd0b87d539/src/bot.py#L82

The part that loads the credentials for the Twitter bot should be rewritten, *so that it loads an **actual** configuration file instead*, rather than a Python module. It should contain the same variables and the same functionality. *We shouldn't import a "module" with 4 variables.*

You can see how a user can load credentials under normal circumstances in the [README file](README.md).

You can find official documentation and instructions in [the Python docs](https://docs.python.org/3/library/configparser.html). Here's a checklist of what you should do;
* Have the bot import `configparser`
* Rewrite the lines 79-87 (as of 8564458).
* Add a sample configuration file inside of the [src](src) folder.
* Replace the instructions in the [README file](README.md) file in **a separate commit.**
* Open a pull request :D

### PR DESCRIPTION
**Pull request by sopneshgandhi**
I have updated the code as per the requirements, kindly review it.

### CODE REVIEW

File: src/bot.py [hunk 1]
```diff
@@ -79,7 +80,12 @@
 elif use_file_variables == True:
     logging.info("Using file variables.")
     try:
-        from credentials import *
+        config = configparser.ConfigParser()
+        config.read('config.txt')
+       	CONSUMER_KEY = config['credentials']['CONSUMER_KEY']
+    	CONSUMER_SECRET = config['credentials']['CONSUMER_SECRET']
+    	ACCESS_KEY = config['credentials']['ACCESS_KEY']
+    	ACCESS_SECRET = config['credentials']['ACCESS_SECRET']
```
Comment (n0toose) [inline src/bot.py]: Mind fixing the indentation over here? This seems to break the Python code.

File: src/config.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[credentials]
+CONSUMER_KEY = <CONSUMER_KEY>
+CONSUMER_SECRET = <CONSUMER_SECRET>
+ACCESS_KEY = <ACCESS_KEY>
+ACCESS_SECRET = <ACCESS_SECRET>
```
Comment (n0toose) [inline src/config.txt]: I get the point of using a placeholder in this case, but I don't think that it's not a good idea to assign values that are incorrect by default.

Comment (n0toose) [inline src/config.txt]: Make sure to remove `config.py` from your commit, because it can be considered as obsolete.

**Review by n0toose — CHANGES_REQUESTED** [review]
Hey, that's some nice work right there! I haven't managed to test it out yet, but there are a few issues that I came across after a quick look.

File: src/config.txt (hunk 2 above)
Comment (sopneshgandhi) [reply src/config.txt]: I understand your point, should I assign a NULL value to the variables initially?

Comment (n0toose) [reply src/config.txt]: Yeah, something like `CONSUMER_KEY =` :D

**Comment by n0toose** [comment]
Hey there, @sopneshgandhi. Excuse me for taking too long with the review, I've just been a bit busy.

This looks good to me, although I don't think that the `""`'s should be there (in `config.txt`), because that would make someone think that they need to put their credentials inside the quotation marks. If they don't remove the quotation marks, the authentication will fail.

I'll test it out at some point in the near future, thanks for your patience.

**Comment by n0toose** [comment]
@sopneshgandhi Hey, could you address the last remaining minor issue with your pull request? Thank you!

**Comment by sopneshgandhi** [comment]
> @sopneshgandhi Hey, could you address the last remaining minor issue with your pull request? Thank you!

I have deleted the file as per the requirment.

File: src/config.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[credentials]
+CONSUMER_KEY = ""
+CONSUMER_SECRET = ""
+ACCESS_KEY = ""
+ACCESS_SECRET = ""
```
Comment (n0toose) [inline src/config.txt]: These quotation marks need to be removed.

File: src/bot.py [hunk 4]
```diff
@@ -79,7 +80,12 @@
 elif use_file_variables == True:
     logging.info("Using file variables.")
     try:
-        from credentials import *
+        config = configparser.ConfigParser()
+        config.read('config.txt')
+       	CONSUMER_KEY = config['credentials']['CONSUMER_KEY']
```
Comment (n0toose) [inline src/bot.py:84]: There's an indentation error over here.

**Review by n0toose — CHANGES_REQUESTED** [review]

File: src/bot.py (hunk 4 above)
Comment (sopneshgandhi) [reply src/bot.py:85]: I am submitting a properly formatted file, I am not getting why it's getting an indentation error after uploading.

> There's an indentation error over here.

I am submitting a properly formatted file, I am not getting why it's getting an indentation error after uploading.

Comment (sopneshgandhi) [reply src/bot.py:84]: I am submitting a properly formatted file, I am not getting why it's getting an indentation error after uploading.

Comment (n0toose) [reply src/bot.py:84]: @sopneshgandhi Maybe you could try editing it with GitHub?

**Comment by n0toose** [comment]
@sopneshgandhi Hey there, sorry for the delay, I wasn't aware that you fixed the issue with your contribution, so I needlessly waited for a new commit until I wondered whether I could fix it by myself-- the issue seems to have been resolved already.

I will test it out later, make any changes if necessary, then merge it.

**Pull request merged**